news/2023/2023-07-27-summer-splash.md

```diff
@@ -0,0 +1,132 @@
+
+[**ksenikust**](https://osu.ppy.sh/users/8938068)'s seriously stylish pippi enchanted **2146** voters with her overwhelming star power and many osu!dachi(tm) branded drinks and accessories. Any resemblances to a tiny squealing Bocchi is purely coincidental.
+
+[![](https://assets.ppy.sh/contests/179/banners/Type-banner.jpg)](https://assets.ppy.sh/contests/179/winners/Type.png)
+
+[**Type**](https://osu.ppy.sh/users/7520534) comes out of left field with an INCREDIBLY cinematic entry, gaining the attention of **2127** voters! Air, earth and sea are the name of the game here with this absolutely beautiful rendition of their signature style that we first saw only last year during the New Beginnings Art Contests. Amazing stuff!
+
+[![](https://assets.ppy.sh/contests/179/banners/ZunK-banner.jpg)](https://assets.ppy.sh/contests/179/winners/ZunK.png)
+
+**2112** voters couldn't help but smash the like button on [**ZunK**](https://osu.ppy.sh/users/1070783)'s piece portraying pippi, Mocha and Aiko trying to nab the perfect summer shot to put in their scrapbook of memories. W-wait, Aiko wasn't ready! Re-do!
+
+[![](https://assets.ppy.sh/contests/179/banners/Novarece-banner.jpg)](https://assets.ppy.sh/contests/179/winners/Novarece.png)
+
+**2082** people waited in line to order from [**Novarece**](https://osu.ppy.sh/users/18728833)'s beachside stall featuring a smoothie-toting pippi expertly multitasking while ordering. This begs the question, if you order a set of beatmap pancakes, do they come in a set? We may never know.
+
+[![](https://assets.ppy.sh/contests/179/banners/tehfire-banner.jpg)](https://assets.ppy.sh/contests/179/winners/tehfire.png)
+
+**2039** voters kneeled before [**tehfire**](https://osu.ppy.sh/users/7082924)'s insanely stylish pastel portrayal of pippi, Mocha, Mari and the Taikonator living out the good life on a beach in perhaps the most painterly piece we've seen yet. What a creation!
+
+[![](https://assets.ppy.sh/contests/179/banners/Amplui-banner.jpg)](https://assets.ppy.sh/contests/179/winners/Amplui.png)
+
+**2012** divers held their breath along with [**Amplui**](https://osu.ppy.sh/users/24820805)'s Mocha to see who'd tap out first. Hint: it's not going to be her, since you'd never want to make that adorable face frown, right?
+
+[![](https://assets.ppy.sh/contests/179/banners/Infinity21_EXE-banner.jpg)](https://assets.ppy.sh/contests/179/winners/Infinity21_EXE.png)
+
+**1809** voters shared a peaceful moment with [**Infinity21_EXE**](https://osu.ppy.sh/users/23873884)'s take on Alisa enjoying a picturesque rest by the seaside. Less is often more!
+
+### peppy's Picks
+
+Wrangling code and configuring lazers is hard work, but rest assured, the Dean "peppy" Herbert has taken some time from his schedule to wield the mighty power of The Pick, as is our (sometimes) tradition. When we asked to describe his choices, he simply opted to unleash a defensive lazer into the ceiling and fled without further explanation.
```

Comment:
either spell laser correctly here or tell people that's not how it's spelt. because i feel like we're doing a disservice at this point 😅

Reply:
if you insist